lib/generators/shopify_app/install/templates/shopify_session_repository.rb

```diff
@@ -1,23 +0,0 @@
-# You should replace InMemorySessionStore with what you will be using
```

Comment:
This is basically just describing the ShopifyApp::Shop concern which is part of the gem now so I think we are covered.